client/src/app/tabs/dmn/DmnEditor.js

```diff
@@ -908,3 +1004,32 @@ function getMigrationDialog() {
     checkboxLabel: 'Do not ask again.'
   };
 }
+
+/**
+ * Check layout whether overview is open.
+ *
+ * @param {Object} props
+ *
+ * @returns {boolean}
+ */
+function isOverviewOpen(props) {
+  return props.layout && props.layout.dmnOverview && props.layout.dmnOverview.open;
+}
+
+/**
+ * Check layout whether overview was opened.
+ *
+ * @param {Object} props
+ * @param {Object} prevProps
+ *
+ * @returns {boolean}
+ */
+function isOverviewOpenend(props, prevProps) {
+  const layout = props.layout,
+        prevLayout = prevProps.layout;
+
+  const open = layout.dmnOverview ? layout.dmnOverview.open : false,
```

Comment:
Why don't we use `isOverviewOpen` here?

Reply:
We can, I added the `isOverviewOpen` function later so I missed that.

Reply:
👍